docs/index.md

```diff
@@ -10,6 +10,8 @@ SPDX-License-Identifier: GPL-3.0-or-later
 [![Nox badge](https://github.com/ansible-community/antsibull-changelog/workflows/nox/badge.svg?event=push&branch=main)](https://github.com/ansible-community/antsibull-changelog/actions?query=workflow%3A%22nox%22+branch%3Amain)
 [![Codecov badge](https://img.shields.io/codecov/c/github/ansible-community/antsibull-changelog)](https://codecov.io/gh/ansible-community/antsibull-changelog)
 
+> Need help or want to discuss the project? See our [Community guide](https://ansible.readthedocs.io/projects/antsibull-changelog/community/) to learn how to join the conversation!
```

Comment:
I don't think this should come **before** the main information for the project itself. Can you please move it before line 27 (i.e. before the start of the next section)?